spec/api/documentation/organizations_api_spec.rb

```diff
@@ -19,6 +19,7 @@
       field :status, 'Status of the organization'
       field :quota_definition_guid, 'The guid of quota to associate with this organization', example_values: ['org-quota-def-guid']
       field :billing_enabled, 'If billing is enabled for this organization', deprecated: true
+      field :default_isolation_segment_guid, 'The guid for the default isolation segment', experimental: true
```

Comment:
We are no longer automatically updating the v2 api docs via these tests. In order for this change to be reflected in the docs, you will have to manually edit them.